datacommons/examples/places.py

```diff
@@ -56,6 +57,11 @@ def main():
   pd_frame = dc.flatten_frame(pd_frame)
   print(pd_frame)
 
+  # Get all population and observation data of Mountain View.
```

Comment:
Maybe move to datacommons/examples/populations.py. See comment below about moving get_pop_obs to populations submodule.